### DISCUSSION TITLE
error TS2339: Property 'order' does not exist on type 'PrismaService'.

### DISCUSSION THREAD

**Discussion by iqbalbals** [prisma/prisma#22541] [Q&A]
### Question

Hello, 
```
src/modules/users/users.service.ts:439:43 - error TS2339: Property 'order' does not exist on type 'PrismaService'.

439     const dataenRoute = await this.prisma.order.count({
                                              ~~~~~

src/modules/users/users.service.ts:454:45 - error TS2339: Property 'm_group_status' does not exist on type 'PrismaService'.

454     const CanceledGroup = await this.prisma.m_group_status.findFirst({
                                                ~~~~~~~~~~~~~~

src/modules/users/users.service.ts:466:44 - error TS2339: Property 'order' does not exist on type 'PrismaService'.

466     const dataCanceled = await this.prisma.order.count({
                                               ~~~~~

src/modules/users/users.service.ts:480:40 - error TS2339: Property 'm_group_status' does not exist on type 'PrismaService'.

480     const RTSGroup = await this.prisma.m_group_status.findFirst({
                                           ~~~~~~~~~~~~~~

src/modules/users/users.service.ts:492:39 - error TS2339: Property 'order' does not exist on type 'PrismaService'.

492     const dataRTS = await this.prisma.order.count({
                                          ~~~~~

src/modules/users/users.service.ts:506:41 - error TS2339: Property 'order' does not exist on type 'PrismaService'.

506     const dataOrder = await this.prisma.order.aggregate({
                                            ~~~~~

src/modules/users/users.service.ts:521:48 - error TS2339: Property 'order' does not exist on type 'PrismaService'.

521     const dataOrderEnRoute = await this.prisma.order.aggregate({
                                                   ~~~~~

src/modules/users/users.service.ts:536:44 - error TS2339: Property 'order' does not exist on type 'PrismaService'.

536     const dataOrderRTS = await this.prisma.order.aggregate({
 ```  

I using nestjs and prisma 
i got error about prisma services, 

i tried to prisma db pull & prisma generate 
and i try to prisma init
schema pull success but when i run the app

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @iqbalbals 👋 

Thank you for raising this question.

A few users who ran into this issue reported that restarting the IDE or restarting the Typescript server in VSCode helped resolve this issue. See the [source](https://stackoverflow.com/questions/69274503/property-does-not-exist-when-i-want-to-use-model-added-in-prisma-schema).

You may also want to check that `PrismaService` is correctly extending `PrismaClient` and that it's being used properly within your service. You would also want to ensure that you've properly registered `PrismaModule` in your NestJS app's module (AppModule or a feature module).

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!

**Comment by CSH111** [comment]
vscode problem in my case

 1.ctrl+shift + p
 2.Typescript: Select Typescript Version...
 3. use Workspace Version (or try select others)
